File: 3-strings/1168.cpp
#include <iostream>
#include <cstring>
#include <vector>
using namespace std;
// 
int main()
{
    int r;
    cin>>r;
    // 
    cin.clear();
    cin.ignore(2147483647,'\n');
    // 
    for(int i=0;i<r;i++){
        string num;
        getline(cin,num);
        int prin=0;
        auto leds=[num]()->int
        { 
            int luz=0;
            for(char j:num){
                int p=j-'0';
                (p==1)?luz+=2:(p==2|p==3|p==5)?luz+=5:(p==4)?luz+=4:(p==6|p==9|p==0)?luz+=6:(p==7)?luz+=3:luz+=7;
            }
            return luz;
        };
        prin=leds();
        cout<<prin<<" leds"<<endl;
    };
    cout<<"\n";
    return 0;
}
    

File: 3-strings/1168.py
for i in range(int(input())):
    led=input()
    luz=0
    for i in led:
        if i=="1":luz+=2
        elif i=='2' or i=='3' or i=='5':luz+=5
        elif i=='4':luz+=4
        elif i=='6'or i=='9'or i=='0':luz+=6
        elif i=='7':luz+=3
        else: luz+=7
    print(luz,'leds')